resources/ts/components/users_admin/UserSection.vue

```diff
@@ -0,0 +1,275 @@
+                !this.user.name ||
+                !this.user.password ||
+                !this.user.password2
+            ) {
+                this.validationError = "Należy uzupełnić wszystkie pola.";
+                return false;
+            }
+
+            if (this.user.password != this.user.password2) {
+                this.validationError = "Hasła powinny być takie same.";
+                return false;
+            }
+
+            if (
+                !this.user.password.match(
+                    /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{6,}$/
+                )
+            ) {
+                this.validationError =
+                    "Hasło powinno składać się z co najmniej 6 znaków, jednej litery i jednej cyfry.";
+                return false;
+            }
+
+            return true;
+        },
+        openDeleteMode(id: number) {
+            this.deleteMode = true;
+            this.user.id = id;
+        },
+        openEditMode(user?: User) {
+            this.clearFrom();
+            this.editMode = true;
+
+            if (user) {
+                this.isEdit = true;
+                this.user.id = user.id;
+                this.user.name = user.name;
+            }
+        },
+        clearFrom() {
+            this.user.name = "";
+            this.user.password = "";
+            this.user.password2 = "";
+        },
+    },
+    created() {
+        this.fetchUsers();
+    },
+    watch: {
+        response() {
+            setTimeout(() => {
+                this.response = "";
+            }, 10000);
+        },
+        validationError() {
+            setTimeout(() => {
+                this.validationError = "";
+            }, 10000);
+        },
+    },
+};
+</script>
+
+<template>
+    <section :class="deleteMode == true || editMode == true ? 'disable' : ''">
+        <div class="action">
+            <h2>Użytkownicy</h2>
+            <p class="response">{{ response }}</p>
+            <button @click="openEditMode()">Dodaj użytkownika</button>
+        </div>
+        <table>
+            <tr>
+                <th>Nazwa</th>
+                <th>Akcje</th>
+            </tr>
+            <tr v-for="user in users" :key="user.id">
+                <td>{{ user.name }}</td>
+                <td>
+                    <font-awesome-icon
+                        class="icon"
+                        icon="fa-solid fa-pen-to-square"
+                        @click="openEditMode(user)"
+                    />
+                    <font-awesome-icon
+                        class="icon"
+                        icon="fa-solid fa-trash"
+                        @click="openDeleteMode(user.id)"
+                    />
+                </td>
+            </tr>
+        </table>
+    </section>
+    <DeleteMode
+        v-if="deleteMode == true"
+        :text="'użytkownika'"
+        :deleteObject="() => deleteUser(user.id)"
+        @close-delete-mode="() => (deleteMode = false)"
+    />
+    <UserForm
```

Comment:
Do poprawy nezewnictwo komponentow

Reply:
tzn? Bardziej coś w stylu 'DeleteModeWindow'?

Reply:
Bardziej coś w stylu
```suggestion
    <delete-mode
        v-if="deleteMode == true"
        :text="'użytkownika'"
        :deleteObject="() => deleteUser(user.id)"
        @close-delete-mode="() => (deleteMode = false)"
    />
    <user-form>
```